<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiClass</class>
 <widget class="QWidget" name="QtGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiClass</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="searchEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="searchButton">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
